params/config.go

```diff
@@ -192,6 +192,11 @@ type ChainConfig struct {
 	Kip103CompatibleBlock *big.Int       `json:"kip103CompatibleBlock,omitempty"` // Kip103Compatible activate block (nil = no fork)
 	Kip103ContractAddress common.Address `json:"kip103ContractAddress,omitempty"` // Kip103 contract address already deployed on the network
 
+	// Randao is an optional hardfork
+	// RandaoCompatibleBlock, RandaoRegistryRecords and RandaoRegistryOwner all must be specified to enable Randao
+	RandaoCompatibleBlock *big.Int        `json:"randaoCompatibleBlock,omitempty"` // RandaoCompatible activate block (nil = no fork)
+	RandaoRegistry        *RegistryConfig `json:"randaoRegistry,omitempty"`        // Registry initial states
```

Comment:
`RegistryAddr` omitted? Regarding this, the registry address would be immutable in the life of the chain, right?

Reply:
The latest [KIP-149 draft](https://github.com/klaytn/kips/pull/150/files) says the Registry must be installed at the fixed address 0x401, though it can change as KIP-149 change.

Reply:
> RegistryAddr omitted? 

I meant that I thought the registry address should have been introduced such as KIP103 address and GovParam contract address. So, I had a concern it was intentionally omitted.

> the registry address would be immutable in the life of the chain, right?

Once its address is fixed in the KIP, it would not be immutable, unlikely that the GovParam adress can be mutable by voting, correct?

Reply:
Yes, it is intentionally omitted to use 0x401 throughout all chains, just like how we use 0x400.
The whole idea of using Registry is to unify system contract management practices. The methods used by KIP103 and GovParam will no longer be practiced.

The Registry address will be immutable in the life of the chain. GovParam and any other system contract addresses can change via modifying the Registry.